core/src/packets/icmp/v4/echo_request.rs

```diff
@@ -4,6 +4,30 @@ use crate::packets::Packet;
 use crate::{Result, SizeOf};
 use std::fmt;
 
+/// Echo Request Message defined in [IETF RFC 792].
+///
+/// ```
+///  0                   1                   2                   3
+///  0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
+/// +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+/// |     Type      |     Code      |          Checksum             |
+/// +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+/// |           Identifier          |        Sequence Number        |
+/// +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+/// |     Data ...
+/// +-+-+-+-+-
+/// ```
+///
+/// Identifier      An identifier to aid in matching Echo Replies
```

Comment:
maybe make these comments same as echo_reply? (specific instead of generic for reply and response)

Reply:
I was emulating our v6 variation, where reply was given a bit of specific info, but it's the true the spec for icmpv4 treats them essentially the same. @drunkirishcoder thoughts?

Reply:
either way is fine, I don't have a particular preference.